src/index.ts

```diff
@@ -188,8 +188,12 @@ class LiskMigrator extends Command {
 			cli.action.start('Creating genesis assets');
 			const createAsset = new CreateAsset(db);
 			const tokenID = getTokenIDLsk();
-			const snapshotHeightPrevious = getHeightPrevSnapshotBlock();
-			const genesisAssets = await createAsset.init(snapshotHeight, snapshotHeightPrevious, tokenID);
+			const prevSnapshotBlockHeight = getHeightPrevSnapshotBlock();
```

Comment:
Optional, but please consider renaming `getHeightPrevSnapshotBlock` to `getPrevSnapshotBlockHeight` instead for naming consistency and clarity.